Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -4851,7 +4851,7 @@ final class BuildPlanTests: XCTestCase {
 
         let yamlContents: String = try fs.readFileContents(yaml)
         let inputs: SerializedJSON = """
-            inputs: ["\(AbsolutePath("/Pkg/Snippets/ASnippet.swift"))","\(swiftGetVersionFilePath.escapedPathString)","\(AbsolutePath("/Pkg/.build/debug/Lib.swiftmodule"))"
+            inputs: ["\(AbsolutePath("/Pkg/Snippets/ASnippet.swift")._nativePathString(escaped: false))","\(swiftGetVersionFilePath.escapedPathString)","\(AbsolutePath("/Pkg/.build/debug/Lib.swiftmodule")._nativePathString(escaped: false))"
         """
         XCTAssertMatch(yamlContents, .contains(inputs.underlying))
```

Comment:
If `inputs` type is changed to plain `String`, that will use string interpolation of `AbsolutePath` that calls `_nativePathString(escaped: false)` automatically.

```suggestion
        let inputs = """
            inputs: ["\(AbsolutePath("/Pkg/Snippets/ASnippet.swift"))","\(swiftGetVersionFilePath.escapedPathString)","\(AbsolutePath("/Pkg/.build/debug/Lib.swiftmodule"))"
        """
        XCTAssertMatch(yamlContents, .contains(inputs))
```